data/refprog/refprog-xml-importer/src/main/java/com/farao_community/farao/data/refprog/refprog_xml_importer/RefProgImporter.java

```diff
@@ -47,23 +47,29 @@ public static ReferenceProgram importRefProg(InputStream inputStream, OffsetDate
         }
         List<ReferenceExchangeData> exchangeDataList = new ArrayList<>();
         document.getPublicationTimeSeries().forEach(timeSeries -> {
-            Country outArea = null;
-            Country inArea = null;
+            ReferenceProgramArea outArea;
+            String outAreaValue = timeSeries.getOutArea().getV();
             try {
-                outArea = (new EICode(timeSeries.getOutArea().getV())).getCountry();
+                new EICode(outAreaValue).getCountry();
+                outArea = new ReferenceProgramArea(outAreaValue);
             } catch (IllegalArgumentException e) {
                 LOGGER.warn("EIC code {} is not mapped to a country. The flow from this area will not be saved.", timeSeries.getOutArea().getV());
+                outArea = new ReferenceProgramArea(timeSeries.getOutArea().getV());
             }
+            ReferenceProgramArea inArea;
+            String inAreaValue = timeSeries.getInArea().getV();
             try {
-                inArea = (new EICode(timeSeries.getInArea().getV())).getCountry();
+                new EICode(inAreaValue).getCountry();
+                inArea = new ReferenceProgramArea(inAreaValue);
             } catch (IllegalArgumentException e) {
                 LOGGER.warn("EIC code {} is not mapped to a country. The flow to this area will not be saved.", timeSeries.getInArea().getV());
+                inArea = new ReferenceProgramArea(timeSeries.getInArea().getV());
             }
-            if (outArea != null || inArea != null) {
+            if (inArea.isVirtualHub() && outArea.isVirtualHub()) {
+                LOGGER.warn("Neither origin ({}) nor extremity ({}) EIC code is mapped to a country. The flow will not be imported.", outAreaValue, inAreaValue);
```

Comment:
I think we should import flows between virtual hubs, it make sense from a functional viewpoint, and I would not be surprise that we have some of these in HVDC modelling.

Cannot we keep the previous if :              `if (outArea != null || inArea != null) {`  ?

Reply:
ok, I was not sure if it made sense. I removed this and adapted a unit test accordingly.